component/Page/Profile/index.tsx

```diff
@@ -181,7 +184,7 @@ export default function ProfilePage(props: PageProps) {
                             <div className="border-t border-gray02" />
                             {/* Card Content */}
                             <div>
-                                <text className="font-normal text-sm">{data?.bio ?? ''}</text>
+                                <LinkableText text={data?.bio}/>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the `<text>` element with the `<LinkableText>` component is done correctly. However, please ensure that the `LinkableText` component handles null or undefined text gracefully, as `data?.bio` could potentially be undefined.
```diff
- <text className="font-normal text-sm">{data?.bio ?? ''}</text>
+ <LinkableText text={data?.bio}/>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->